### ISSUE TITLE
.NET/C# (server and client)

### ISSUE THREAD

**Issue by alanctkc** [smallstep/hello-mtls#22]
This could probably be split into multiple issues. I'm not sure what the options are here.

**Comment by alanctkc** [comment]
@sourishkrout Related to #17, or is this still a separate issue (ie. .NET can terminate TLS on its own without IIS)?

**Comment by sourishkrout** [comment]
@alanctkc it supports TLS w/o IIS when I add the intermediate cert to the root bundle (hacky). do we have any other docs that suggest doing that to get around lack of intermediate support?

**Comment by alanctkc** [comment]
Yes, we do! We have to add the intermediate to the root in MySQL clients, and we make mention of it on the server side, even though we don't bundle the intermediate there.

- https://smallstep.com/hello-mtls/doc/client/mysql-client
- https://smallstep.com/hello-mtls/doc/server/mysql

**Comment by sourishkrout** [comment]
Great. I will model `.NET`'s after this referencing the open github bug.
